Cloudinary-pwa-kit/packages/pwa-cloudinary/app/components/cloudinary-image-gallery/index.jsx

```diff
@@ -44,6 +44,14 @@ const CloudinaryImageGallery = ({ size, cloudinaryImageGallery = {} }) => {
         imageUrls = cloudinaryImageGallery?.images?.imageURLs
     }
     const imageUrl = imageUrls ? imageUrls[selectedIndex] : null
+
+    useEffect(() => {
+        if (imageUrl?.isResponsive) {
+            window.cldObj = window.cldObj || window?.cloudinary?.default?.Cloudinary?.new({cloud_name: cloudinaryImageGallery.cloudName || cloudinaryImageGallery}); // eslint-disable-line no-undef
```

Comment:
Please initiate the gallery with the CNAME logic (cld-319)

Reply:
the 3rd and 4th questions marks are not needed

Reply:
The Image Gallery is not rendering on this component, it's on another component (RenderCloudinaryGalleryWidget). Would you like for us to update there?

Reply:
Apologize I wasn't clear enough. There is no need to move the widget initiation to this location.
Here we define the widget initiation options object, and the options object needs to include the CNAME as was added in cld-319

Reply:
you mean we need to implement the same thing as we did in CLD-319?

Reply:
yes, we need to have the same configuration object definition

Reply:
Can we please have a separate ticket for that as we have for the core cartridge, the CLD-319, this ticket and PR have totally separate context, a separate ticket will help to track the implementation.

Reply:
sure

Reply:
are you going to create the ticket or do you want me to create that?

Reply:
I created cld-326